api_types.ts

```diff
@@ -494,6 +494,15 @@ export interface CommonPackFormulaDef<T extends ParamDefs> {
    * so an end user must have both sets of permissions.
    */
   readonly extraOAuthScopes?: string[];
+
+  /**
+   * TODO(patrick): Without this set, should we allow all or only DEFAULT?
```

Comment:
For backwards compatibility, I think we have to take it to mean at a minimum default if present + system if present.

Could choose a different default if auxiliary auth types are defined but that may be confusing.

May be simplest to take "everything" as the default.

Reply:
I'm leaning towards "everything"